modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -26,38 +28,20 @@
       context 'when not provided' do
         it 'returns a 401 error code' do
           with_okta_user(scopes) do
-            get path
+            get all_claims_path
             expect(response.status).to eq(401)
           end
         end
       end
     end
 
-    context 'forbidden access' do
-      context 'when current user is not the target veteran' do
-        context 'when current user is not a representative of the target veteran' do
-          it 'returns a 403' do
-            with_okta_user(scopes) do |auth_header|
-              expect_any_instance_of(ClaimsApi::V2::ApplicationController)
-                .to receive(:user_is_target_veteran?).and_return(false)
-              expect_any_instance_of(ClaimsApi::V2::ApplicationController)
-                .to receive(:user_represents_veteran?).and_return(false)
-
-              get path, headers: auth_header
-              expect(response.status).to eq(403)
-            end
-          end
-        end
-      end
-    end
-
```

Comment:
Do we not want this spec?

Reply:
whoops, I just accidentally deleted that when copy-pasting around. I'll add it back in